roadside-forms-frontend/frontend_web_app/src/components/Dashboard/Dashboard.js

```diff
@@ -78,15 +80,15 @@ export const Dashboard = () => {
         setStaticDataLoaded(true);
 
         try {
-          db.vehicles.bulkPut(vehicleData.data);
-          db.vehicleStyles.bulkPut(vehicleStyleData.data);
-          db.vehicleColours.bulkPut(vehicleColourData.data);
-          db.provinces.bulkPut(provinceData.data);
-          db.impoundLotOperators.bulkPut(impoundData.data);
-          db.jurisdictions.bulkPut(jurisdictionData.data);
-          db.countries.bulkPut(countryData.data);
-          db.cities.bulkPut(cityData.data);
-          db.agencies.bulkPut(agencyData.data);
+          await db.vehicles.bulkPut(vehicleData.data);
```

Comment:
Nit: These could probably be grouped into one big "await" so that the processes can run concurrently within one await